featurebyte/feast/utils/registry_construction.py

```diff
@@ -414,10 +460,42 @@ def create_feast_feature_services(
             feature_services.append(feature_service)
         return feature_services
 
+
+class FeastRegistryBuilder:
+    """
+    Class for constructing the Feast Registry.
+    """
+
+    @staticmethod
+    def _create_repo_config(
+        project_name: str, online_store: Optional[OnlineStoreModel], registry_file_path: str
+    ) -> RepoConfig:
+        online_store_config: Optional[Dict[str, Any]] = None
+        if online_store:
+            online_store_for_type = {
+                OnlineStoreType.REDIS: RedisOnlineStoreConfig,
+                OnlineStoreType.MYSQL: MySQLOnlineStoreConfig,
+            }
+            online_store_config_class = online_store_for_type[online_store.details.type]
+            online_store_config = online_store_config_class(
+                **online_store.details.dict(by_alias=True, exclude={"credential": True})
+            ).dict(by_alias=True)
```

Comment:
Is the online store config created without the credentials information? Another comment mentioned that feast apply will be attempting to make connection to online store, will there be an issue without the credentials?

Reply:
Good catch, will update this to use `featurebyte.feast.model.online_store.get_feast_online_store_details`.